demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -92,27 +93,114 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
     return docker_python_path
 
 
-def update_dotenv(values: Dict[str, str], quote_mode="never"):
-    env_vars = dotenv.dotenv_values(DOTENV_PATH)
+def update_dotenv(file_path: Path, values: Dict[str, str], quote_mode="never"):
+    """
+    Configure the .env file with the given values. Generates the file if it doesn't exist.
+
+    Args:
+        file_path (Path): The path to the .env file
+        values (Dict[str, str]): The values to set
+        quote_mode (str): The quote mode to use. Defaults to "never"
+    """
+    file_path.touch(exist_ok=True)
+    env_vars = dotenv.dotenv_values(file_path)
     env_vars.update(values)
     for key, value in env_vars.items():
         if value:
             dotenv.set_key(DOTENV_PATH, key, value, quote_mode=quote_mode)
         else:
-            logger.warning(f"empty value for {key}, not setting it")
+            logger.warning(f"Empty value for '{key}'. Skipping...")
```

Comment:
```suggestion
            logger.warning(f"Empty value for '{key}'. Not setting it.")
```

Reply:
To my understanding, these are pre-existing `.env` vars set by the user. Not ones we add (we skip parsing of existing values that are empty).

Reply:
Those arey emtpy values, like `DEMISTO_SDK_BASE_URL=`. We insert the pre-existing values to the dictionary and treat them in the same way.

Reply:
@MichaelYochpaz